aiida_vasp/parsers/node_composer.py

```diff
@@ -0,0 +1,146 @@
+"""A Node Composer for Aiida data nodes."""
+# pylint: disable=no-member
+# Reason: pylint erroneously complains about non existing member 'get_quantity', which will be set in __init__.
+
+from aiida_vasp.utils.aiida_utils import get_data_class
+from aiida_vasp.utils.delegates import delegate_method_kwargs, Delegate
+from aiida_vasp.parsers.quantity import ParsableQuantities
+
+NODES_TYPES = {
+    'parameter': ['outcar-volume', 'outcar-energies', 'outcar-efermi', 'symmetries'],
```

Comment:
How would we do this if we want to use say these things from `vasprun` by default, but drop down into `outcar` in case `vasprun` does not exists. Guess this is handled as usual by the alternatives? E.g. as long as `outcar-volume` is defined as an alternative in the `vasprun` parser config, then it is fine and it changes automatically?

Reply:
That's how it should work, I think.

Reply:
Right, then I am assuming this works.

Reply:
The `NODE_TYPES` dictionary would be indeed the place to put the quantities you would like to have added to the node by default. So if it should be only quantities from 'vasprun.xml' you would have to replace 
```
'parameter': ['outcar-volume', 'outcar-energies', 'outcar-efermi', 'symmetries'] 
```
with
```
'parameter': ['volume', 'energies', 'fermi_level', 'symmetries']
```
I kept for the moment the old defaults, because changing the defaults was not in the scope of this PR.

The quantities you list here will be checked first, whether they can be parsed. If they can't be parsed and they have an 'alternatives' list, the quantities from the 'alternatives' list will be checked. Here is a general example how it is supposed to work.
```
# The main quantity:
'volume': {
    'inputs': [],
    'name': 'volume',
    'prerequisites':  [],
    }

# And the alternative quantity
'outcar-volume': {
    'inputs': [],
    'name': 'volume',
    'prerequisites':  [],
    }
```
With this setup 'outcar-volume' will be recognised as an alternative for 'volume', because its name is 'volume'. In case that 'volume' can't be parsed, 'outcar-volume' will be checked instead. The alternatives for the main quantity will be set automatically.
If it the order in which alternatives are checked is important, you can define the 'alternatives' for the main quantity explicitly:
```
'volume': {
    'inputs': [],
    'name': 'volume',
    'prerequisites':  [],
    'alternatives': ['outcar-volume']
    }
```
In this 'outcar-volume' will always be checked before any other alternative, that might be appended automatically to that list.